Documentation/Images.md

```diff
@@ -21,3 +21,31 @@ StyleSheet.default.mutate(imageLoader: MyImageLoader())
 ```
 
 The `ImageLoader` protocol only requires one method. The `completion` method must be invoked, even if the loading fails. It also must be invoked on the main thread.
+
+# Scaling images
+
+By default, NativeMarkKit will not resize the images it downloads, but render them at their natural size. This isn't always desirable, so nativeMarkKit provides a customization point to allow whatever kind of scaling the app wants.
+
+The image sizing behavior is attached to the `StyleSheet`, and can be overridden there. To provide a custom image sizer, the app needs to implement the `ImageSizer` protocol (or use a built-in implementation), then mutate any `StyleSheet`s it wants to use its custom image sizer.
+
+```Swift
+import NativeMarkKit
+
+struct MyImageSizer: ImageSizer {
+    func imageSize(_ urlString: String?, image: NativeImage, lineFragment: CGRect) -> CGSize {
```

Comment:
Seems `urlString` seems useless to me since when this delegate is called the image should be already available.

Reply:
`urlString` is to allow for dynamic resizing. i.e. the sizer can look at the url and determine how it wants to scale. For example, an app might inject local app icons via an `icon://` scheme, which it wants to scale to line height. But it might also want to scale regular `http://` schema images by width. Having the url allows it to do that.